main/lib/src/TransportTCP.cc

```diff
@@ -593,6 +628,7 @@ namespace eudaq {
             EUDAQ_THROW_NOLOG(LastSockErrorString("SocketClient Error (" + to_string(LastSockError()) + ")"));
           }
           else if (result > 0){
+			
             m_buf.append(result, buffer);
             while (m_buf.havepacket()) {
               m_events.push(TransportEvent(TransportEvent::RECEIVE, m_buf, m_buf.getpacket()));
```

Comment:
Thanks, that doesn't belong into this PR, but I will pick it and push to v1.4-dev.